### PR TITLE
Update Android SDK v3.0.0 release notes

### PR DESCRIPTION
**Pull request by vonageCI**
Android SDK v3.0.0 notes added to `release-notes.md`.

### CODE REVIEW

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md [hunk 1]
```diff
@@ -6,6 +6,41 @@ navigation_weight: 0
...
+### Added
+
+- Added `NexmoMemberSummary` returned by `conversation.getMembers(pageSize, order, listener)` (paginated), representing a subset member information.
+- Added `memberEvent.getInvitedBy()` that represents the inviter name, if exists.
+- Added `NexmoEventEmbeddedInfo` to all events returned by `event.getEmbeddedInfo()` and containing the `NexmoUser` linked to the event.
+- Added `conversation.getMember(memberId, listener)` returning the member given its identifier.
+
+### Enhancements
+
+- Improve the javadoc documentation.
```
Comment (VMadalin) [inline _documentation/en/client-sdk/sdk-documentation/android/release-notes.md:20]: Seems javadoc is not present on the spell check, maybe we should consider to simplify with:
```
- Improve the documentation.
```

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md [hunk 2]
```diff
@@ -6,6 +6,41 @@ navigation_weight: 0
 
 # Release Notes
 
+## Version 3.0.0 - Jun 1, 2021
+
+### Added
+
+- Added `NexmoMemberSummary` returned by `conversation.getMembers(pageSize, order, listener)` (paginated), representing a subset member information.
+- Added `memberEvent.getInvitedBy()` that represents the inviter name, if exists.
```
Comment (VMadalin) [inline _documentation/en/client-sdk/sdk-documentation/android/release-notes.md:14]: Seems the inviter isn't present on the spell check maybe we should replace it with something else, any suggestion?

**Review by abdulajet — APPROVED** [review]
:shipit:

File: _documentation/en/client-sdk/sdk-documentation/android/release-notes.md (hunk 2 above)
Comment (abdulajet) [reply _documentation/en/client-sdk/sdk-documentation/android/release-notes.md:14]: there is a `.spelling` file you can update with anything causing the linter to fail

**Pull request merged**